Providers/nxOMSAutomationWorker/automationworker/worker/hybridworker.py

```diff
@@ -23,16 +24,19 @@ def decorated_func(*args, **kwargs):
             try:
                 # ensure required file / cert exists
                 func(*args, **kwargs)
-            except (JrdsAuthorizationException,
-                    InvalidFilePermissionException,
+            except (JrdsAuthorizationException):
+                tracer.log_worker_safe_loop_terminal_exception(traceback.format_exc())
+                time.sleep(workerpollingfrequency.get_jrds_get_sandbox_actions_polling_freq()) #polling frequency as per the value received from headers of GetSandboxActions
```

Comment:
we can remove this, the sleep below is outside of try catch, so I belive it already workers for authorixation exception too.